### PR TITLE
Updated Comparison to Excel documentation with code example of read_excel and to_excel for #38990

### PR DESCRIPTION
**Pull request by bsun94**
- This is my first time contributing - please let me know if any other detail is needed! Essentially I just swapped the order of the "Excel" and "CSV" paragraphs in the data importing section, so that the examples given for to_excel and read_excel would flow from the CSV example without needing repeated code/codeblocks.

### CODE REVIEW

File: doc/source/getting_started/comparison/comparison_with_spreadsheets.rst [hunk 1]
```diff
@@ -125,6 +118,27 @@ would be:
...
+
+.. code-block:: python
+
+    tips.to_excel("./tips.xlsx")
+
+Should you wish to subsequently access the data in the ``tips.xlsx`` file, you can read it into your module using
+
+.. code-block:: python
+
+    tips_df = pd.read_excel("./tips.xlsx", header=None)
```
Comment (afeld) [inline doc/source/getting_started/comparison/comparison_with_spreadsheets.rst:138]: By default, `to_excel()` has `header=True` and `index=True`, so the following would give a more accurate round-trip:

```suggestion
    tips_df = pd.read_excel("./tips.xlsx", index_col=0)
```

**Review by afeld — APPROVED** [review]
Just the one note. Otherwise, 👍 from me. Nice work!

**Comment by afeld** [comment]
Oh, sorry, one other thing: for future commits, make sure to use [these prefixes](https://pandas.pydata.org/docs/dev/development/contributing.html#committing-your-code). I think that can be fixed by whomever merges doing a [squash](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits), so don't worry about changing it (unless you want to — [instructions](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message)).

**Comment by bsun94** [comment]
Thank you Aidan, Jeff - and thank you Aidan for guiding me on my first ever pandas PR! hopefully more to come!

**Comment by jreback** [comment]
thanks @bsun94

**Pull request merged**